static/index.js

```diff
@@ -91,4 +91,52 @@ document.addEventListener("DOMContentLoaded", event => {
     }
   });
 
+  var darkSwitch = document.querySelector('input[name=darkMode]');
+  const prefersDarkScheme = window.matchMedia("(prefers-color-scheme: dark)");
+  const prefersLightScheme = window.matchMedia("(prefers-color-scheme: light)");
+
+  darkSwitch.addEventListener('change', function (event) {
+    // toggles darkmode when user taps switch
+    if (localStorage.getItem("darkModelocal") == "dark") {
+      document.documentElement.classList.toggle("dark");
```

Comment:
Let's add the `force` param for toggle in both light and dark mode. If `darkModeLocal` is dark, and somehow the document is already on dark mode, it won't be a problem even if we toggle.